subprojects/test-runner/device-provider/src/main/kotlin/com/avito/android/runner/devices/internal/kubernetes/KubernetesReservationClient.kt

```diff
@@ -134,20 +134,20 @@ internal class KubernetesReservationClient(
                 .distinctBy { it.metadata.name }
                 .consumeEach { pod ->
                     launch(CoroutineName("boot-pod-${pod.metadata.name}")) {
-                        pod.bootDevice(serialsChannel)
+                        pod.bootAndSendDevice(serialsChannel)
                     }
                 }
             logger.debug("initializeDevices finished")
         }
     }
 
-    private suspend fun Pod.bootDevice(serialsChannel: Channel<DeviceCoordinate>) {
+    private suspend fun Pod.bootAndSendDevice(serialsChannel: Channel<DeviceCoordinate>) {
```

Comment:
I think it's better to extract `send` part and return `Result<DeviceCoordinate.Kubernetes>`